src/inference/Inference.jl

```diff
@@ -138,6 +139,8 @@ const TURING_INTERNAL_VARS = (internals = [
     "nom_step_size",
     "tree_depth",
     "is_adapt",
+    "walker_id",
+    "iteration"
```

Comment:
I don't think these are needed if the samples of each walker end up in a separate chain in a `Chains` object? Due to these internal parameter names I assumed you're concatenating samples in a different way, but judging from your comment below you want to handle it in the same way as in AdvancedMH.

Reply:
```suggestion
```

Reply:
Yeah, this was leftover code when I was spitting everything out into one chain.